Silence errors when trying to read thermal data which is not present.

In VMs (where we run) there will generally not be any thermal data.
Whilst the actual thermal calling code is careful to only log at debug
to handle this, the actual uint64 reading routine reports an error
rather than logging at debug.  Use the newly introduced read_uint64_at_level
routine to bypass this.
--- a/plugins/in_node_exporter_metrics/ne_cpu_linux.c
+++ b/plugins/in_node_exporter_metrics/ne_cpu_linux.c
@@ -112,19 +112,23 @@
         entry = mk_list_entry(head, struct flb_slist_entry, _head);
 
         /* Core ID */
-        ret = ne_utils_file_read_uint64(ctx, ctx->path_sysfs,
+        ret = ne_utils_file_read_uint64_at_level(ctx, ctx->path_sysfs,
                                         entry->str,
-                                        "topology", "core_id",
-                                        &core_id);
+                                        "topology",
+                                        "core_id",
+                                        &core_id,
+                                        FLB_LOG_DEBUG);
         if (ret != 0) {
             continue;
         }
 
         /* Physical ID */
-        ret = ne_utils_file_read_uint64(ctx, ctx->path_sysfs,
+        ret = ne_utils_file_read_uint64_at_level(ctx, ctx->path_sysfs,
                                         entry->str,
-                                        "topology", "physical_package_id",
-                                        &physical_package_id);
+                                        "topology",
+                                        "physical_package_id",
+                                        &physical_package_id,
+                                        FLB_LOG_DEBUG);
         if (ret != 0) {
             continue;
         }
